WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -1580,11 +1581,14 @@ class ReaderPostDetailFragment : ViewPagerFragment(),
     private fun handleDirectOperation() = when (directOperation) {
         DirectOperation.COMMENT_JUMP, DirectOperation.COMMENT_REPLY, DirectOperation.COMMENT_LIKE -> {
             viewModel.post?.let {
-                ReaderActivityLauncher.showReaderComments(
-                    activity, it.blogId, it.postId,
-                    directOperation, commentId.toLong(), viewModel.interceptedUri,
-                    DIRECT_OPERATION.sourceDescription
-                )
+                val maybeActivity: FragmentActivity? = activity
+                maybeActivity?.let { fragmentActivity ->
+                    ReaderActivityLauncher.showReaderComments(
+                        fragmentActivity, it.blogId, it.postId,
+                        directOperation, commentId.toLong(), viewModel.interceptedUri,
+                        DIRECT_OPERATION.sourceDescription
+                    )
+                }
```

Comment:
np: I will make the same comment here? Just checking `context?.let {` looks simpler to me, as you also did in the `ReaderDiscoverFragment`.

Reply:
Thanks, updated 👍